.gitignore

```diff
@@ -1,2 +1,3 @@
 test/db
 Gemfile.lock
+gemfiles/*.gemfile.lock
```

Comment:
how about we go for the delete lock on travis method ... not a big fan of removing those lock files, makes coming back to old projects a [pain](http://grosser.it/2015/08/14/check-in-your-gemfile-lock) :)